src/pages/SendTokens/SendTokens.js

```diff
@@ -1,7 +1,59 @@
-import React from "react";
+import React, { useEffect, useState } from 'react';
+import apiClient from '../../utils/apiClient';
+import { Loader } from '../../components/UI/components/Loader/Loader';
+import { Grid, Paper } from '@mui/material';
+import ErrorMessage from '../../components/UI/components/ErrorMessage/ErrorMessage';
+import InfoCircle from './InfoCircle';
 
 const SendTokens = () => {
-  return <div style={{ marginTop: "9.4vh" }}>SendTokens</div>;
+
+  const [walletList, setWalletList] = useState({});
+  const [errorMessage, setErrorMessage] = useState(null);
+  const [isLoading, setIsLoading] = useState(true);
+
+
+  //when the page is first rendered
+  useEffect(() => {
+    const loadWallets = async () => {
+      try {
+        // TODO: default limit
+        const wallets = await apiClient.get('/wallets?limit=200');
```

Comment:
Instead of using the default wallet by getting all wallets, you should get wallet by id by using this API: "{{baseUrl}}/wallets/{{id}}" as sender wallet will be selected from the dropdown

Reply:
I figured that the dropdown would be populated by getting all the wallets and then displaying. How will the dropdown be populated with options?

Reply:
Yes, you are right, we don't need extra requests